examples/aws-lambda/index.ts

```diff
@@ -0,0 +1,21 @@
+import "reflect-metadata";
+import { buildSchema } from "../../src";
+import { RecipeResolver } from "./recipe-resolver";
+import { GraphQLSchema } from "graphql";
+import { APIGatewayProxyHandlerV2 } from "aws-lambda";
+import { ApolloServer } from "apollo-server-lambda";
+
+let cachedSchema: GraphQLSchema | null = null;
+let cachedServer: ApolloServer | null = null;
+
+export const handler: APIGatewayProxyHandlerV2 = async (event, context, callback) => {
+  // build TypeGraphQL executable schema
+  cachedSchema ??= await buildSchema({
+    resolvers: [RecipeResolver],
+  });
+
+  // Create GraphQL server
+  cachedServer ??= new ApolloServer({ schema: cachedSchema });
+
+  return cachedServer.createHandler({})(event, context, callback);
+};
```

Comment:
This is the only difference in this `aws-lambda` example, compared to the base one.

I think it might be more valuable to create `Recipes` section in docs, where we can put some simple snippets like this.
I don't think that it's as complicated as typegoose or typeorm integration that it deserves own example.